.github/workflows/e2e.yml

```
@@ -14,6 +14,7 @@ on:
 
 jobs:
   run-ci:
+    if: vars.RUN_E2E == 'true'
```

You can also use something like 
```suggestion
    if: ${{ github.repository == 'ubicloud/ubicloud' }}
```

---

I thought it'd be nice if non-`ubicloud/ubicloud` repos could set up E2E tests without applying a patch